scripts/jest/setup.js

```diff
@@ -20,4 +20,14 @@ import 'jest-axe/extend-expect';
 import 'raf/polyfill';
 import registerRequireContextHook from 'babel-plugin-require-context-hook/register';
 
+// The below is a workaround to the problem were calling resetModules causes react to be required twice.
+// further details can be found here: https://github.com/jestjs/jest/issues/8987#issuecomment-584898030
+let mockActualReact;
+jest.doMock('react', () => {
```

Comment:
Ah nice! I remember seeing a similar issue in the past with `isolateModules` so we do the same in BpkModalV2 https://github.com/Skyscanner/backpack/blob/main/packages/bpk-component-modal/src/BpkModalV2/BpKModal-test.tsx#L27
I guess it should be ok to do this across all tests, we could probably remove the one in modal too?

Reply:
Yup, let me know if you want me to add that to this PR

Reply:
Yes, please if you don't mind! 😄

Reply:
I've tried to remove the instances of `isolateModule` in the test you referenced, which most of the tests are happy with but some are failing so I think it's going to require a bit more time for someone to work out the nuances of that test file.